pages/common/mv.md

```diff
@@ -33,4 +33,4 @@
 
 - Specify target directory (convenient in situations when the target directory has to be the first argument):
```

Comment:
This is a quick and dirty example of what I'm after:
```suggestion
- Specify target directory so that you can use external tools to gather movable files:
```